packages/angular_devkit/build_angular/src/angular-cli-files/models/build-options.ts

```diff
@@ -23,7 +24,8 @@ export interface BuildOptions {
   outputPath: string;
   resourcesOutputPath?: string;
   aot?: boolean;
-  sourceMap?: boolean;
+  sourceMap?: SourceMap;
```

Comment:
The build options at this point should be fully normalized which should be handled at least one level up from here.

Reply:
The thing is that in a number of cases, I need to know if the sourceMaps is for scripts or styles. 

Example: 
https://github.com/angular/angular-cli/pull/13062/files#diff-fdb0776b9f2db468444818966705634cR50
https://github.com/angular/angular-cli/pull/13062/files#diff-ce7fb1412b327df10292e336d2e96e16R20

You propose to create two options? `scriptsSourceMap` and `styleSourceMap` to normalize it and ?

Reply:
There's really two different sets of options here.  The "boolean or object" options are a user interface level concern.  But at the level here the options object should be fully actionable and unambiguous.